src/mame/seta/seta.cpp

```diff
@@ -6422,10 +6422,18 @@ static INPUT_PORTS_START( zingzip )
 	PORT_BIT( 0x0002, IP_ACTIVE_LOW, IPT_UNKNOWN  ) // no coin 2
 	PORT_BIT( 0x0004, IP_ACTIVE_LOW, IPT_SERVICE1 )
 	PORT_BIT( 0x0008, IP_ACTIVE_LOW, IPT_TILT     )
-	PORT_BIT( 0x0010, IP_ACTIVE_LOW, IPT_UNKNOWN  )
-	PORT_BIT( 0x0020, IP_ACTIVE_LOW, IPT_UNKNOWN  )
-	PORT_BIT( 0x0040, IP_ACTIVE_LOW, IPT_UNKNOWN  )
-	PORT_BIT( 0x0080, IP_ACTIVE_LOW, IPT_UNKNOWN  )
+	PORT_CONFNAME( 0x0010, 0x0010, DEF_STR( Unknown ) )
+	PORT_CONFSETTING(      0x0010, DEF_STR( Off ) )
+	PORT_CONFSETTING(      0x0000, DEF_STR( On ) )
+	PORT_CONFNAME( 0x0020, 0x0020, DEF_STR( Unknown ) )
+	PORT_CONFSETTING(      0x0020, DEF_STR( Off ) )
+	PORT_CONFSETTING(      0x0000, DEF_STR( On ) )
+	PORT_CONFNAME( 0x0040, 0x0040, DEF_STR( Unknown ) )
+	PORT_CONFSETTING(      0x0040, DEF_STR( Off ) )
+	PORT_CONFSETTING(      0x0000, DEF_STR( On ) )
+	PORT_CONFNAME( 0x0080, 0x0080, "Title" )           // Not listed in manual
+	PORT_CONFSETTING(      0x0080, "With Chinese" )
+	PORT_CONFSETTING(      0x0000, "Without Chinese" ) // Without 真真急炮
```

Comment:
Maybe make this more descriptive.  Could you call this setting “Title Language”, and call the options “English and Chinese” and “English only”?